### ISSUE TITLE
"Palaeozoic Echinoidea" subclass

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#246]
https://data.catalogueoflife.org/dataset/2284/taxon/856H3
from worms.

Should we blacklist "Palaeozoic" as a name part or can it be a genuine name?

**Comment by yroskov** [comment]
As it present in the source.

It's not a unique case:

![image](https://user-images.githubusercontent.com/28574252/112313150-8972da80-8c75-11eb-9aa5-5587bf928ac3.png)

There is no choice, names and taxa will stay in CoL as they are supplied by the provider. Changes are possible in master database on provider's side.

**Comment by mdoering** [comment]
we tried hard to eliminate incertae sedis taxa. Is it a good idea to give that up? An accepted name _Echinoid trace fossils (ichnotaxa)_?

**Comment by yroskov** [comment]
It's not possible to change "manually" all cases in automated WoRMS checklists via clearingouse software. It will create mess in the classification soon, or will create a huge load of work on us with every update. Repeat: data might be improved on data provider side only. CoL can just re-publish what we get from WoRMS.

Number of "spoiled" taxa has been increased dramatically after we allowed taxa of intermediate ranks and synonymy for taxa above species be present in CoL.

**Comment by mdoering** [comment]
I agree it is unmanageable to chase every non scientific name and do a manual replacement or ignore decision on each on them individually. But it should be doable to collect patterns of such taxa that we then filter out during syncs. We don't need to filter out all right away. But slowly we will be able to reduce their numbers as we learn or get feedback.

And for cases when there is no pattern we could apply a decision that just ignored this taxon (which we apparently not yet have - block will exclude also all descendants)

**Comment by yroskov** [comment]
Anger against unauthorized data modification made by the current software is growing among data providers. For example, have a look on comment here: https://github.com/CatalogueOfLife/testing/issues/61 

Another example is a big problem with programmatically assigned trinomials.
